samples/net/mqtt_publisher/src/main.c

```diff
@@ -61,15 +61,69 @@ struct mqtt_client_ctx {
 	void *publish_data;
 };
 
-/* This is the network context structure. */
-static struct net_context *net_ctx;
-
 /* The mqtt client struct */
 static struct mqtt_client_ctx client_ctx;
 
 /* This routine sets some basic properties for the network context variable */
-static int network_setup(struct net_context **net_ctx, const char *local_addr,
-			 const char *server_addr, u16_t server_port);
+static int network_setup(void);
+
+#if defined(CONFIG_MQTT_LIB_TLS)
+
+#include "test_certs.h"
+
+/* TLS */
+#define TLS_SNI_HOSTNAME "localhost"
+#define TLS_REQUEST_BUF_SIZE 1500
```

Comment:
So I was just wondering here why would we need to have 2500 byte long mbedtls setting if we always read max 1500 bytes long data (size of the request buf).